docs/recipes/close-account.md

```diff
@@ -0,0 +1,32 @@
+# Close Account
+
+In accounting, a _closing entry_ calculates the net debit or credit balance for an account and then
+credits or debits this balance respectively, to zero the account's balance and move the balance to
+another account.
+
+### Example
+
+Given a set of accounts:
+
+| Account | Debits Pending | Debits Posted | Credits Pending | Credits Posted | Flags                            |
+| ------: | -------------: | ------------: | --------------: | -------------: | -------------------------------- |
+|     `A` |              0 |            10 |               0 |             20 | `debits_must_not_exceed_credits` |
+|     `B` |              0 |            30 |               0 |              5 | `credits_must_not_exceed_debits` |
+|     `C` |              0 |             0 |               0 |              0 |                                  |
+
+The "closing entries" for accounts `A` and `B` are:
+
+| Debit Account   | Credit Account | Amount     | Amount (recorded) | Flags              | Notes               |
+| --------------: | -------------: | ---------: | ----------------: | ------------------ | ------------------- |
+|             `A` |            `C` | `2^64 - 1` |                10 | `balancing_debit`  | (close account `A`) |
+|             `C` |            `B` | `2^64 - 1` |                25 | `balancing_credit` | (close account `B`) |
+
+(Pass `maxInt(u64)` as the `Transfer.amount` so that the application does not need to know (or query) the balance prior to closing the account).
```

Comment:
I think we discussed rather using `0` as the null value, like we do elsewhere, e.g. for posting transfers. This saves the developer from having to think in terms of type size minus 1.

Reply:
Thanks for the reminder, I forgot about that!

Reply:
Updated.